libmscore/keysig.cpp

```diff
@@ -535,6 +566,11 @@ bool KeySig::setProperty(Pid propertyId, const QVariant& v)
                         return false;
                   setShowCourtesy(v.toBool());
                   break;
+            case Pid::KEYSIG_MODETYPE:
+                  if (generated())
+                        return false;
+                  //setModeType(v);
```

Comment:
This could be the reason why it's not set by the inspector maybe ?

Reply:
It is commented out because it doesn't compile ;-)

Reply:
no known conversion from 'const QVariant' to 'Ms::KeyMode'